### PR TITLE
Update local storage docs for v2.0 provisioner

### PR DESCRIPTION
**Pull request by msau42**
Wait until mount detection change is in, and v2 is tagged.

### CODE REVIEW

File: local-volume/provisioner/CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,20 @@
+# v2.0.0
+This version is incompatible and has breaking changes with v1!
+* Remove default config, a configmap is now required.
+* Configmap data is changed from json to yaml syntax.
+* All local volumes must be mount points.  For directory-based volumes, a
+  bind-mount must be done in order for the provisioner to discover them.
+* Detected capacity is rounded down to the nearest GB.
+* Mount propagation is required to discover new mount points after the
```
Comment (dhirajh) [inline local-volume/provisioner/CHANGELOG.md]: Does it make sense to refer to https://kubernetes.io/docs/concepts/storage/volumes/#mount-propagation or mention k8s 1.8 has it?

**Review by dhirajh — APPROVED** [review]
LGTM, just one cosmetic comment.

File: local-volume/provisioner/CHANGELOG.md (hunk 1 above)
Comment (msau42) [reply local-volume/provisioner/CHANGELOG.md]: Done.  The top level README contains specific instructions on how to enable mount propagation and in what version

**Comment by wongma7** [comment]
this is ready for merge, the change is already in kube right?

**Comment by msau42** [comment]
Yup this can be merged

**Pull request merged**